composeml/label_times/description.py

```diff
@@ -3,28 +3,37 @@
 
 def describe_label_times(label_times):
     """Prints out label info with transform settings that reproduce labels."""
-    if label_times.label_name is not None and label_times.is_discrete:
+    target_column = label_times.target_columns[0]
```

Comment:
If we run this function on a `LabelTimes` object that has multiple target columns defined, the output lists only a single column. I think this should be fine once we have run `.select()` on the label times object. It seems like we should list all the possible target columns in the Settings output if none have been selected.

Reply:
A single target is currently required to be selected before the label description can be made ([source](https://github.com/FeatureLabs/compose/blob/selects_single_target/composeml/label_times/object.py#L173-L174)). In a separate PR, we do plan to list all targets in the label description if none have been selected. This is being tracked in #122.